### DISCUSSION TITLE
More information about architecture

### DISCUSSION THREAD

**Discussion by Asanoci** [ourownstory/neural_prophet#414] [General]
Hello,
I am new to this library, and i would like to understand better how it internally works.
Is there any paper in which we could see that ?

For example, i would like to know if:
The predictions generated by a FFNN ? Or are FFNN only used to calculate the effects of added regressors ?

Thank you ! :)

**Comment by Asanoci** [comment]
Does anyone have any information ? :)

**Comment by gkumarg** [comment]
Wondering if you have already taken a look at the Notes folder? There are couple of documents there.

**Comment by ourownstory** [comment]
Hi @Asanoci I hope, https://github.com/ourownstory/neural_prophet/blob/master/notes/NeuralProphet_Introduction.pdf addresses your question?
